docs/source/api/server_server_capabilities.rst

```diff
@@ -222,4 +222,4 @@ Response Structure
 				"level": "success"
 			}
 		]
-	}
\ No newline at end of file
+	}
```

Comment:
I can't quite figure out what was changed on this line - the file does have a terminating newline doesn't it? It appears to, but otherwise idk what changed.

Reply:
It did not have a terminating newline.

Reply:
Yep, it's a newline. Old file didn't have it, new one does. Github just doesn't show it very well.

```
$ git show fa512c0dc29be4ce766c279469c3d403197912aa
@@ -222,4 +222,4 @@ Response Structure
                                "level": "success"
                        }
                ]
-       }
\ No newline at end of file
+       }
```